addons/hr_presence/models/hr_employee.py

```diff
@@ -21,16 +21,18 @@ class Employee(models.AbstractModel):
     # Stored field used in the presence kanban reporting view
     # to allow group by state.
     hr_presence_state_display = fields.Selection([
-        ('present', 'Present'),
-        ('absent', 'Absent'),
-        ('to_define', 'To Define')])
+        ('1_to_define', 'Absence To Check'),
+        ('2_absent', 'Justified Absence'),
+        ('3_present', 'Present')])
```

Comment:
Why did you change the technical terms?

Reply:
The spec asked for this particular order and the prefix ensures the ordering in the view.

Reply:
I wouldn't change it. This requires a migration script. 

Reordering the keys should change the order for new databases (to confirm but I think that's the case), and old database remain that way.